src/platform/forms/save-in-progress/SaveInProgressIntro.jsx

```diff
@@ -241,22 +256,24 @@ class SaveInProgressIntro extends React.Component {
     const content = (
       <div>
         {!this.props.buttonOnly && this.getAlert(savedForm)}
-        <FormStartControls
-          resumeOnly={this.props.resumeOnly}
-          isExpired={isExpired}
-          messages={this.props.messages}
-          startText={this.props.startText}
-          startPage={startPage}
-          formId={this.props.formId}
-          returnUrl={this.props.returnUrl}
-          migrations={this.props.migrations}
-          prefillTransformer={this.props.prefillTransformer}
-          fetchInProgressForm={this.props.fetchInProgressForm}
-          removeInProgressForm={this.props.removeInProgressForm}
-          prefillAvailable={prefillAvailable}
-          formSaved={!!savedForm}
-          gaStartEventName={this.props.gaStartEventName}
-        />
+        {(this.props.user.login.currentlyLoggedIn || this.props.buttonOnly) && (
```

Comment:
Do we want the `this.props.buttonOnly` here? What I'm specifically thinking about is if we want the "continue to the form..." button on the bottom of the form intro page to still be shown for logged-out users (shown below). This is probably a question for @Samara-Strauss and/or @BBarone1 but I'd think we'd want to remove it to further discourage people from filling out the form without logging in first.

![image](https://user-images.githubusercontent.com/20728956/52130478-29310280-25ef-11e9-8542-90d25ed0a0ee.png)

Reply:
Can we also change this second button to "Sign in to Start Your Application"?

Reply:
I think that conditionally changing the label on that button based on the user being signed in or not might be out of scope for this ticket. But I _was_ wondering if this button on the bottom of the intro page should be shown at all if the user isn't logged in? They'd still be able to continue on to form without logging in but they'd need to new text link on the bottom of the alert.

Reply:
If we can't make this button "Sign in and start...", I'd hide it.

Reply:
the more important issue that I should have realized 30 minutes ago is that I don't think we can change the _functionality_ of this button. At least not easily. It will still just direct to the form without forcing you to sign in first. So for now at least it should be removed for logged-out users.

Reply:
Good catch. So it sounds like we'll stop showing the button at the bottom for now.